dumbbells/q_functions.py

```diff
@@ -70,23 +70,30 @@ def __init__(
         self.arch = arch
         self.gamma = gamma
         self.optimizer = (
-            torch.optim.RMSprop(self.arch.parameters()) if optim is None else optim
+            torch.optim.RMSprop(self.arch.parameters(), lr=1e-5)
```

Comment:
We should make this a variable so that we can change while fine tuning our model

Reply:
Nice catch! I didn't realise I left it like so